app/bundles/ReportBundle/Entity/Report.php

```diff
@@ -137,6 +167,30 @@ public static function loadMetadata(ORM\ClassMetadata $metadata)
             ->columnName('settings')
             ->nullable()
             ->build();
+
+        $builder->createField('isScheduled', 'boolean')
+            ->columnName('is_scheduled')
+            ->build();
+
+        $builder->createField('scheduleUnit', 'string')
```

Comment:
`'string'` can be replaced with Doctrine's constants like here: https://github.com/mautic/mautic/blob/staging/app/bundles/LeadBundle/Entity/Import.php#L181

This syntax can be also simplified with `addNullableField` method.

Reply:
Fixed, thanks